pkg/apis/elasticsearch/v1/autoscaling.go

```diff
@@ -52,6 +57,10 @@ type AutoscalingPolicy struct {
 // +kubebuilder:object:generate=false
 type AutoscalingSpec struct {
 	AutoscalingPolicySpecs AutoscalingPolicySpecs `json:"policies"`
+
+	// SyncPeriod is the period at which to synchronize and poll the Elasticsearch autoscaling API.
+	SyncPeriod *metav1.Duration `json:"syncPeriod"`
```

Comment:
Should it simply be called `pollingFrequency` or `pollingPeriod`? If I read `sync` I wonder what it is being synchronized with what.